### PR TITLE
Update to Swift 2.2 syntax

### PR DESCRIPTION
**Pull request by phchang**

### CODE REVIEW

**Comment by phchang** [comment]
The CI failure seems to be the same as this issue: https://github.com/travis-ci/travis-ci/issues/3986

**Comment by tonystone** [comment]
@phchang See my comments in this thread concerning the same issue https://github.com/travis-ci/travis-ci/issues/5743#issuecomment-213851125

Can you update the TraceLog travis.yml to be the same structure as GeoFeatures 1 here https://github.com/tonystone/geofeatures/blob/master/.travis.yml

You don't have to include the global: - COCOAPODS_DISABLE_DETERMINISTIC_UUIDS=1 but the rest should pretty much be exactly the same accept for the name of the project and scheme.

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,20 @@
 language: objective-c
 xcode_workspace: Example/TraceLog.xcworkspace
 xcode_scheme: TraceLog-Example
```
Comment (tonystone) [inline .travis.yml]: @phchang can you switch out `xcode_workspace:` and `xcode_scheme:` here for `os: osx` as it is in GeoFeatures 1?  These are redundant and don't need to be here, just more for us to maintain and keep in sync.

**Comment by tonystone** [comment]
Approved, all clear to merge @phchang.

**Pull request merged**